src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Draft.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@using SFA.DAS.Encoding
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort.DraftViewModel
+@inject IEncodingService EncodingService
+
+@{
+    ViewData["Title"] = "Draft";
+}
+
+<div class="govuk-grid-row">
+
+        <div class="govuk-grid-row">
+            <div class="govuk-width-container">
+                <h1 class="govuk-heading-xl">Draft apprentice details</h1>
+
+                <table class="govuk-table das-table--responsive">
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header das-table-cell-width-25">Training provider</th>
+                            <th scope="col" class="govuk-table__header das-table-cell-width-25">Reference</th>
+                            <th scope="col" class="govuk-table__header das-table-cell-width-25 govuk-table__header--numeric">Number of apprentices</th>
+                            <th scope="col" class="govuk-table__header govuk-table__header--numeric das-table-cell-width-25"></th>
+                        </tr>
+                    </thead>
+                    <tbody class="govuk-table__body">
+
+                        @foreach (var cohort in Model.Cohorts)
+                        {
+                            var cohortReference = EncodingService.Encode(cohort.CohortId, EncodingType.CohortReference);
+
+                            <tr class="govuk-table__row">
+                                <td class="govuk-table__cell" data-label="Training provider">@cohort.ProviderName</td>
+                                <td class="govuk-table__cell" data-label="Reference">@cohortReference</td>
+                                <td class="govuk-table__cell govuk-table__cell--numeric" data-label="Number of apprentices">@cohort.NumberOfDraftApprentices</td>
+                                <td class="govuk-table__cell govuk-table__cell--numeric" data-label="">
+                                    @Html.ActionLink("Details", "Details", "Cohort", new { @Model.AccountHashedId, cohortReference }, new { @class = "govuk-link", id = "return-link" })
```

Comment:
We should probably not use an Id property repeatedly, and this seems a strange Id to use in any case. We could instead add a css class "details-link" or similar as useful for automation tests.